ibm_catalog.json

```diff
@@ -0,0 +1,250 @@
+            {
+              "key": "ibmcloud_api_key",
+              "required": true
+            },
+            {
+              "key": "prefix",
+              "required": true,
+              "description": "The prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
+            },
+            {
+              "key": "cluster_id",
+              "type": "string",
+              "default_value": "",
+              "display_name": "Cluster",
+              "required": true,
+              "custom_config": {
+                  "type": "cluster_var",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment"
+              }
+            },
+            {
+              "key": "cluster_resource_group_id",
+              "type": "string",
+              "default_value": "",
+              "display_name": "Cluster resource group",
+              "required": true,
+              "custom_config": {
+                  "type": "resource_group",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment",
+                  "config_constraints": {
+                      "identifier": "rg_id"
+                  }
+              }
+            },
+            {
+              "key": "is_vpc_cluster",
+              "required": true
+            },
+            {
+              "key": "cloud_monitoring_enabled"
+            },
+            {
+              "key": "cloud_monitoring_access_key"
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace"
+            },
+            {
+              "key": "cloud_monitoring_agent_tags"
+            },
+            {
+              "key": "cloud_monitoring_agent_tolerations"
+            },
+            {
+              "key": "cloud_monitoring_endpoint_type"
+            },
+            {
+              "key": "cloud_monitoring_instance_region",
+              "display_name": "Region",
+              "custom_config": {
+                "type": "vpc_region",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "generationType": "2"
+                }
+              }
+            },
+            {
+              "key": "cloud_monitoring_metrics_filter"
+            },
+            {
+              "key": "cloud_monitoring_secret_name"
+            },
+            {
+              "key": "cluster_config_endpoint_type"
+            },
+            {
+              "key": "cluster_data"
+            },
+            {
+              "key": "cloud_logs_ingress_endpoint"
+            },
+            {
+              "key": "cloud_logs_ingress_port"
```

Comment:
I think I would force require this too, and have it come after `cloud_logs_ingress_endpoint`